components/back-link/README.md

```diff
@@ -25,8 +25,6 @@ With custom click handler
 ### Properties
 Prop | Required | Default | Type | Description
 :--- | :------- | :------ | :--- | :----------
- `children` |  | 'Back' | string | Text that will appear in the back link
```

Comment:
i think we should leave this in - otherwise it's not entirely clear how one sets the text